common/lib/kconfig.ts

```diff
@@ -124,6 +124,8 @@ const normalizedSoloNftUrl = (chainId: number, nft: NftType): string => {
 
 // CONSTANT
 const ipfsGateway = "https://ipfs.io/ipfs/";
+const SWARM_GATEWAY = "https://api.gateway.ethswarm.org";
+const SWARM_GATEWAY_BZZ = SWARM_GATEWAY + "/bzz/";
```

Comment:
define only one const : 
`const SWARM_GATEWAY = config.swarmGateway`

you can make a function to get the other const when needed :
`swarmServer(SWARM_GATEWAY) => "https://api.gateway.ethswarm.org"`